go/test/endtoend/backup/vtbackup/backup_only_test.go

```diff
@@ -54,8 +55,7 @@ func TestTabletInitialBackup(t *testing.T) {
 	//    - list the backups, remove them
 
 	vtBackup(t, true)
-	backups := countBackups(t)
-	assert.Equal(t, 1, backups)
+	verifyBackupCount(t, shardKsName, 1)
```

Comment:
can this be replaced with `cluster.VerifyBackupCount`?

Reply:
Inside vtbackup tests for all VerifyBackupCount, listBackups, RemoveAllBackup it uses `vtctl `instead of `vtctld` as used in clusterUtils methods. We tried to replace it with vtctld but none of the configurations worked.

Reply:
ok